components/chat/ChatFeed.vue

```diff
@@ -188,6 +198,15 @@ export default {
   },
 
   methods: {
+        insertEmoji(emoji) {
+    if (!this.postText) {
+      this.postText = emoji + " ";
+    } else {
+      this.postText = this.postText + " " + emoji + " ";
```

Comment:
@Nidz-Web3 If I observed correctly, the emoji package already adds spaces by default, so extra space should not be needed. Try to remove the spaces and see how it looks. If it looks okay, then we don't need these extra spaces.